### DISCUSSION TITLE
Opening links in a new tab or window?

### DISCUSSION THREAD

**Discussion by h-enk** [gethyas/hyas#191] [General]
I'd like to follow current web practice (as much as possible) and [Let the web be the web](https://www.tempertemper.net/blog/opening-links-in-a-new-tab-or-window-is-better-avoided#let-the-web-be-the-web) (as much as possible).

> [..] isn't the general practice to open links leading to a new website in a new tab? If my users click on an external link then I'm losing them by directing them away from my site. I've rarely seen it done and I would rarely recommend it.

I think that's currently not the case. See e.g. [US to lift Covid travel ban for vaccinated passengers from UK and most of EU](https://www.theguardian.com/us-news/2021/sep/20/us-travel-ban-lifted-uk-eu-flights-allowed-vaccinated-latest-covid-restrictions) — the _Dame Karen Pierce_ link, or [Web Quality Assurance: From User Requirements To Web Risk Management](https://www.smashingmagazine.com/2021/09/journey-into-web-quality-assurance/) — the _book about web quality assurance_ link.

## Resources

- [Opening links in a new tab or window is better avoided](https://www.tempertemper.net/blog/opening-links-in-a-new-tab-or-window-is-better-avoided)
- [When to Open Links in a New Tab](https://meiert.com/en/blog/links-in-new-tabs/)
- [G200: Opening new windows and tabs from a link only when necessary](https://www.w3.org/TR/WCAG20-TECHS/G200.html)

**Comment by timtorres** [comment]
Okay I'm onboard with this, thanks for sending the references and examples. I removed that function but am keeping the hook to open downloadable files in a new tab, I don't think that would go into Hyas though.

**Reply by h-enk** [reply]
> I removed that function but am keeping the hook to open downloadable files in a new tab, I don't think that would go into Hyas though.

Correct.
